tests/test_algorithm.py

```diff
@@ -1202,9 +1179,10 @@ def handle_data(context, data):
 
 class TestAlgoScript(WithLogger,
                      WithDataPortal,
+                     WithSimParams,
                      ZiplineTestCase):
-    SIM_PARAMS_END = None
```

Comment:
I am just avoiding the num_days constructor because it doesn't play as nicely with `WithDefaultDateBounds`